lang/main.json

```diff
@@ -128,7 +128,7 @@
         "startVideoMuted": "Everyone starts hidden",
         "selectCamera": "Camera",
         "selectMic": "Microphone",
-        "selectAudioOutput": "Audio output",
+        "selectAudioOutput": "Speakers",
```

Comment:
audio output feels better than speakers to me here, i worry calling this speakers will be confusing

Reply:
Design confirms "Speakers."

Reply:
we're gonna need more discussion on this.

Reply:
Emil doesn't want it changed so we're not changing it. He can discuss with design and the copy writer if they bring it up.